op-chain-ops/deployer/pipeline/opchain.go

```diff
@@ -62,6 +62,7 @@ func DeployOPChain(ctx context.Context, env *Env, intent *state.Intent, st *stat
 						BasefeeScalar:          1368,
 						BlobBaseFeeScalar:      801949,
 						L2ChainId:              chainID.Big(),
+						StartingAnchorRoots:    []byte{},
```

Comment:
Is this value constant? If so, we should hardcode it rather than requiring is to be in the intent. If it's in the intent users will have to specify it prior to deploying every chain.

Reply:
I think for permissioned chains it can be an arbitrary constant, based on [this convo], but should confirm with @Inphi 

(https://discord.com/channels/1244729134312198194/1273519241076670526/1285673518494650482)
<img width="900" alt="image" src="https://github.com/user-attachments/assets/89fc74c2-8642-4eb0-98f4-b9f9b17e0d40">
<img width="878" alt="image" src="https://github.com/user-attachments/assets/62fb28cd-f666-4553-946b-00d95a4c2987">

Reply:
This value is an array of `struct AnchorRoot { GameType, OutputRoot}`, so the _correct_ value will be different for each chain, but because the anchor state is part of the [cyclic dependency](https://github.com/ethereum-optimism/design-docs/pull/79), we won't actually know what the correct value is when deploying a new OP Chain. 

However it must be non-zero because per @Inphi in discord (I've replaced the link he gave with a permalink):

> Actually the starting anchor should be non-zero. Otherwise, the permissioned game will fail this [check](https://github.com/ethereum-optimism/optimism/blob/c21d0c5fa04faa116ef9572f5f9b36d3e03f3a36/packages/contracts-bedrock/src/dispute/FaultDisputeGame.sol). 

So, in order to get this over the line, we may need to simply hardcode some nonsense initial output roots (ie. `0xdead....dead`). That's presumably safe as no meaningful proof would be likely to be discovered which enables a fraudulent withdrawal.